PWGHF/HFL/TableProducer/electronSelectionWithTpcEmcal.cxx

```diff
@@ -251,6 +310,7 @@ struct HfElectronSelectionWithTpcEmcal {
         m20MatchEmcCluster = emcCluster.m20();
         timeEmcCluster = emcCluster.time();
         cellEmcCluster = emcCluster.nCells();
+        elecrtonMatchId = matchTrack.globalIndex();
```

Comment:
is the globalIndex of "matchTrack" same as "track"? If so why not  use the globalIndex of track, so that its the same for TPC+EMCal or TPC+TOF?

Reply:
yes globalIndex of "matchTrack" same as "track" and we can use the globalIndex of track.